src/server.rs

```diff
@@ -49,6 +49,9 @@ impl<D: Sync + Send + 'static> Server<D> {
             server.keep_alive(timeout);
         }
 
-        server.handle(arc)
+        match thread_count {
```

Comment:
Perhaps we should be adding our own calculation here as the default instead of hypers, so we can keep our docs in sync? Or else we'll just have to provide a link to hyper-docs for the calculation?